### ISSUE TITLE
Notification for filled out survey

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#13]
## User story
1. As a {HR employee}
2. I want / need {to get a notification as soon as the DI-owner has filled out the survey and added the required information in Lissi Wallet (which has been requested via self attested attributes in a Proof request)}
3. So that {I can request information from DI-owners}

## Acceptance criteria
* notification on dashboard ( a "news"-section with a table) displayed to HR-employee as soon as proof request (with self attested attributes) has been answered in Lissi Wallet
* table row contains information: "Received update on proof request XY" --> links to the proof request where the detailed update (with values for self attested attributes) can be seen
* If possible, give information in the table on excact date and time when answer has been typed in in Lissi wallet

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
